scripts/csv_importer.py

```diff
@@ -0,0 +1,398 @@
+        "id": generate_id(),
+        "owner_id": owner_id,
+        "type": "create",
+        "logged_at": get_current_time(),
+        "object_id": object_id,
+        "object_label": object_name,
+        "object_class": object_class
+    }
+    insert_statement = (
+        "INSERT INTO "
+        "koi_log (id, owner_id, type, logged_at, object_id, object_label, object_class) "
+        "values  (%(id)s, %(owner_id)s, %(type)s, %(logged_at)s, %(object_id)s, %(object_label)s, %(object_class)s)"
+    )
+    cursor.execute(insert_statement, log_data)
+
+    return log_data["id"]
+
+
+def parse_args() -> dict:
+    """
+    Parses the command line arguments
+    :return: dictionary containing the arguments
+    """
+    parser = argparse.ArgumentParser(description="Import data from a csv file to Koillection database")
+    parser.add_argument("-f", "--csv_file", type=str, required=True, help="Path to the csv file")
+    parser.add_argument("-F", "--compose_file", type=str, help="Path to the docker compose file")
+
+    parser.add_argument("-n", "--name_column", type=str, required=True,
+                        help="CSV column containing item names")
+    parser.add_argument("-p", "--private_fields", type=str, nargs='+', default=[],
+                        help="Column names that will be made private")
+    parser.add_argument("-s", "--skip_fields", type=str, nargs='+', default=[],
+                        help="Column names that will be skipped")
+    parser.add_argument("-c", "--collection", type=str, required=True,
+                        help="Name of the collection to import data into."
+                             "A new collection will be created if it does not exist.")
+    parser.add_argument("-u", "--user", type=str,
+                        help="Koillection user that will become the owner of the newly created collection"
+                             "(not required when there is only one user)")
+
+    parser.add_argument("--skip_empty_columns", action="store_true",
+                        help="Removes all columns that don't have a value in any item")
+    parser.add_argument("--skip_empty_fields", action="store_true",
+                        help="Removes fields that don't have a value (works per-item)")
+
+    parser.add_argument("--host", type=str, help="Database host ip")
+
+    args = vars(parser.parse_args())
+
+    args["private_fields"] = [field_name.lower() for field_name in args["private_fields"]]
+    args["skip_fields"] = [field_name.lower() for field_name in args["skip_fields"]]
+
+    return args
+
+
+def load_environment_variables(compose_file: str) -> dict:
+    """
+    Loads environmental variables from the koillection service, from the provided compose file
+    :param compose_file: path to the compose file
+    :return: dictionary containing the environmental variables and their values
+    """
+
+    with open(compose_file, 'r') as file:
+        compose = yaml.safe_load(file)
+
+    environment = {}
```

Comment:
`return {key: value for item in compose[][][] for key, value in item.split("=")`?
Please consider🙄

Reply:
This solution is definitely shorter, but for me it's a lot less readable

Reply:
Ok, sure. I wrote this commit as we have similar code earlier.